inception-external-search-elastic/src/main/java/de/tudarmstadt/ukp/inception/externalsearch/elastic/traits/ElasticSearchProviderTraitsEditor.html

```diff
@@ -53,6 +53,14 @@
             <input wicket:id="objectType" type="text" class="form-control"/>
           </div>
         </div>
+        <div class="form-group" wicket:enclosure="resultSize">
+          <label class="col-sm-2 control-label">
+            <wicket:message key="resultSize"/>
+          </label>
+          <div class="col-sm-10">
+            <input wicket:id="resultSize" type="number" min="1" max="25000" class="form-control"/>
```

Comment:
`type`, `min` and `max` are not required here since they should be added by Wicket according to the settings you make in the Java code.

Reply:
removed `min` and `max`. 
removing `type` results in an error that is thrown in the method `org.apache.wicket.MarkupContainer.renderNext`, so I left it in.